process.ts

```diff
@@ -121,7 +122,7 @@ function extractHitInto(sourceFiles: SourceFiles, sourcePath: string) {
 }
 
 export function processSourceMaps(
-  bundleSourceMaps: string[],
+  patterns: string[],
```

Comment:
let's keep this interface the same, and instead move the glob sync work into index.ts.